src/validators.rs

```diff
@@ -43,12 +50,22 @@ impl Default for RGBContext {
     fn default() -> Self {
         Self {
             closed_method: "tapret1st".to_string(),
+            current_network: String::new(),
```

Comment:
Would it make sense to default to this being Option::None, since that's the type on RgbInvoice ?

Reply:
Yes, we can work with option. I will change.